### PR TITLE
get all .md reports at once

### PR DESCRIPTION
**Pull request by Nastradamus**

### CODE REVIEW

File: check [hunk 1]
```diff
@@ -541,6 +541,107 @@ is_in_recovery() {
...
+
+  ###### Checking the existence of commands #####
+
+  # timeout
+  if ! $(which gtimeout >/dev/null 2>&1) && ! $(which timeout >/dev/null 2>&1); then
+    err "Can't find the 'timeout' executable. Please install it:"
+    if [[ "${KERNEL_NAME}" = "Darwin" ]]; then
+      err "${OS_NAME}: 'brew install coreutils'"
+    elif [[ "${KERNEL_NAME}" = "Linux" ]]; then
+      err "Ubuntu or Debian GNU/${OS_NAME}: 'apt-get install coreutils'"
```
Comment (NikolayS) [inline check]: `sudo apt-get ...`?

and `RHEL/CentOS: 'sudo yum install ...'` ?

File: check [hunk 2]
```diff
@@ -541,6 +541,107 @@ is_in_recovery() {
...
+    fi
+    return 2
```
Comment (NikolayS) [inline check]: why `2` here? maybe it's worth to document the meaning of every returned value?

**Comment by NikolayS** [comment]
This looks awesome!

One thing -- it looks like the current code doesn't check `jq` well:
```
$ ./check -h db2.imgdata.ru --project test -U nevada_adm -e 1
[2018-12-05T11:53:51-0500]
[2018-12-05T11:53:51-0500] ########## Perform checks for host 'db2':
[2018-12-05T11:53:51-0500]
[2018-12-05T11:53:51-0500] ========== Perform check: ==========
[2018-12-05T11:53:51-0500] ./resources/checks/A001_system_info.sh
./resources/checks/A001_system_info.sh: line 78: jq: command not found
[2018-12-05T11:53:55-0500] ========== End of check ===========
[2018-12-05T11:53:55-0500] WARNING: check 'A001' failed with error ^^^
[2018-12-05T11:53:55-0500]
[2018-12-05T11:53:55-0500] ========== Perform check: ==========
[2018-12-05T11:53:55-0500] ./resources/checks/A002_pgversion.sh
[2018-12-05T11:53:55-0500] ========== End of check ===========
[2018-12-05T11:53:55-0500] check 'A002' is finished
./check: line 775: jq: command not found
```

**Comment by Nastradamus** [comment]
Fixed all commented things.

**Pull request merged**